integration-tests/parse_files.sh

```diff
@@ -5,6 +5,9 @@ set -o nounset
 
 readonly CWD="$PWD"
 readonly TESTS_DIR="integration-tests/Dockerfiles"
+BLACKLIST="./Dockerfiles/dockerfiles/nylas/sync-engine/Dockerfile"
+BLACKLIST=$BLACKLIST" ./Dockerfiles/docker-images/OracleWebLogic/samples/12212-domain/Dockerfile"
+BLACKLIST=$BLACKLIST" ./Dockerfiles/docker-images/OracleWebLogic/samples/12213-domain/Dockerfile"
```

Comment:
The over 3 files have instructions that while `docker build` can parse just fine, they are documented as if they should not be working at all.